normandy/recipes/models.py

```diff
@@ -85,6 +87,13 @@ def arguments(self):
     def arguments(self, value):
         self.arguments_json = json.dumps(value)
 
+    @property
+    def current_approval_request(self):
+        try:
+            return self.approval_requests.get(active=True)
```

Comment:
I think it might be possible to have more than one approval request, which isn't handled here.